app/reducers/history.ts

```diff
@@ -8,9 +8,11 @@ const initialState = initialAppState.history;
 const updateHistory = ( state, payload ) => {
     if ( payload.url ) {
         const historyState = { ...state };
-        const date = new Date().toLocaleDateString();
+        const d = new Date();
+        const date = d.toLocaleDateString();
         const url = makeValidAddressBarUrl( payload.url );
-        const timeStamp = new Date().toLocaleTimeString();
+        const timeStamp = `${d.getUTCHours()}:${d.getUTCMinutes()}:${d.getSeconds()}`;
```

Comment:
just `new Date().getTime()`. Anything else is formatting to be done by frontend.